code/modules/chemistry/tools/patches.dm

```diff
@@ -370,79 +370,6 @@
 	medical = 1
 	initial_reagents = "synthflesh"
 
-/obj/item/patch_stack
```

Comment:
These weren't used anywhere and had some unique storage interfacing code that I didn't wanna untangle, so I just removed them.